keyboards/kingly_keys/ave/config.h

```diff
@@ -0,0 +1,50 @@
+ /* 
+ Copyright 2020 Garret Gartner
+  
+ This program is free software: you can redistribute it and/or modify 
+ it under the terms of the GNU General Public License as published by 
+ the Free Software Foundation, either version 2 of the License, or 
+ (at your option) any later version. 
+  
+ This program is distributed in the hope that it will be useful, 
+ but WITHOUT ANY WARRANTY; without even the implied warranty of 
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the 
+ GNU General Public License for more details. 
+  
+ You should have received a copy of the GNU General Public License 
+ along with this program.  If not, see <http://www.gnu.org/licenses/>. 
+ */ 
+
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0x4B4B // "KK"
+#define PRODUCT_ID      0x1225
+#define DEVICE_VER      0x0011
+#define MANUFACTURER    Kingly-Keys
+#define PRODUCT         "The Ave."
```

Comment:
Is this meant to be in ""?

Reply:
I started doing that after my compiles started failing from some punctuation in the product/description.

Reply:
You might be able to fix it by replacing the . with `\x2E`. It won't look pretty in the file, but at least it won't be quoted in the device list.

Reply:
okay ill change it.  

just out of curiosity though, could it also be modified with using the backslash syntax or is that not applicable here?

Reply:
as in `\.` ?

Reply:
Just tried compiling `6ball:default` with a . added to the end of `PRODUCT`, it seems to have worked without `\x2E` or `\.`.
![image](https://user-images.githubusercontent.com/4781841/97455672-34fc5500-198c-11eb-942f-0aa4df5083d8.png)